src/Dfe.Spi.GiasAdapter.Functions/LearningProviders/GetLearningProvider.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Dfe.Spi.GiasAdapter.Application.LearningProviders;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Dfe.Spi.GiasAdapter.Functions.LearningProviders
+{
+    public class GetLearningProvider
+    {
+        private const string FunctionName = nameof(GetLearningProvider);
+
+        private readonly ILearningProviderManager _learningProviderManager;
+        private readonly ILogger _logger;
+
+        public GetLearningProvider(ILearningProviderManager learningProviderManager, ILogger logger)
```

Comment:
Got a question re: the common logging around injecting here. Will chat to you in a bit!